src/resolvers-cassandra/Edges/queries.js

```diff
@@ -13,29 +13,17 @@ const { trackEvent } = require('../../clients/appinsights/AppInsightsClient');
  */
 function terms(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
-    if (!args) return reject('No args specified');
-
-    const { fromDate, toDate } = parseFromToDate(args.fromDate, args.toDate);
-
     const query = `
-    SELECT topics
-    FROM fortis.mentionedtopics
-    WHERE pipelinekey = ?
-    AND externalsourceid = ?
-    AND event_time <= ?
-    AND event_time >= ?
+    SELECT topic
+    FROM fortis.watchlist
     `.trim();
 
     const params = [
-      toPipelineKey(args.sourceFilter),
-      'all',
-      toDate,
-      fromDate
     ];
 
     cassandraConnector.executeQuery(query, params)
     .then(rows => {
-      const keywords = makeSet(rows.map(row => row.topics), topic => topic);
+      const keywords = makeSet(rows, row => row.topic);
```

Comment:
Rather than `const keywords` maybe it would be clearer to use `const topics` so there's one term for topic rather than a topic's name either being `keyword` or `topic`?